### PR TITLE
[Ready]Fixes portable connector keeping air

### LINKED ISSUE #13700: Connector keeping gases with nothing connected to it
So I was doing more fusion stuff and I was trying to fill up a canister with the stuff at a "normal" pressure. I had a passive gate set up to do so, but before I could even turn it on the canister connected to it instantly filled up to the absurdly high fusion pressures. I tried it without a pump or anything connected to the connector, and the same thing happened.
sybil hut romeo 2015-12-16 100746
Before I wrenched in the canister
sybil hut romeo 2015-12-16 100756
After I wrenched it in

At first I just unwrenched/rewrenched the connector and left the pump turned off, but it still filled up with gas. Then I tried to unwrench/rewrench the pump and the gasses still got in. It kept happening until I had both the pump and the connector unwrench then put back into place

### PR DESCRIPTION
**Pull request by MrsTonedOne**
OH GOD I HOPE THIS DOESN'T BREAK SOMETHING.

It shouldn't they link directly into the pipenet.

fixes #13700

### CODE REVIEW

File: code/ATMOSPHERICS/components/unary_devices/portables_connector.dm [hunk 1]
```diff
@@ -7,6 +7,10 @@
 	var/obj/machinery/portable_atmospherics/connected_device
 	use_power = 0
 	level = 0
+	
+/obj/machinery/atmospherics/components/unary/portables_connector/New()
+	..()
+	airs[1].volume = 0
```
Comment (RemieRichards) [inline code/ATMOSPHERICS/components/unary_devices/portables_connector.dm]: you can't do this in Byond.

also ```AIR1``` is the define to use.

Comment (duncathan) [inline code/ATMOSPHERICS/components/unary_devices/portables_connector.dm]:
```
var/datum/gas/mixture/air_contents = AIR1
air_contents.volume = 0
```

**Comment by tkdrg** [comment]
@MrStonedOne

**Comment by tkdrg** [comment]
@MrStonedOne

**Comment by duncathan** [comment]
god dammit mso i wrote the fixed code for you

**Comment by MrStonedOne** [comment]
Ready

**Pull request merged**